RetroBar/Themes/System.xaml

```diff
@@ -1,5 +1,6 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    x:Class="RetroBar.SystemTheme"
```

Comment:
I don't think this is needed any more.

Reply:
I was afraid I was missing something left to remove, now it's done!